pom.xml

```diff
@@ -1017,7 +1013,7 @@
                   <version>[1.7,)</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>[3.0.3,)</version>
```

Comment:
`maven-os-plugin`, used in gcp-io, requires at least Maven 3.1. As long as we're bumping the dependency, I propose we go to 3.3, which was [released March 2015](https://maven.apache.org/docs/history.html).